mautic/mautic
# Builder preview button

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | x
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Cloning the preview input/button from details.html.php to builder.html.php for email and page

#### Steps to test this PR:
1. Select an email from the email list view
2. Click "edit" from the details view
3. Click "open builder" from the edit view
4. Observe the builder panel the public preview URL should be populated correctly and the button should open the preview URL in a new page
5. Close email
6. Select a page from the page list view
7. Repeat steps 2-4 for the page.
